crates/pallet-humanode-offences/src/lib.rs

```diff
@@ -0,0 +1,70 @@
+//! A substrate pallet containing the humanode offences handler logic.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use frame_support::{
+    sp_std::prelude::*,
+    traits::{Get, StorageVersion},
+};
+pub use pallet::*;
+use sp_staking::offence::{Offence, OffenceError, ReportOffence};
+
+/// The current storage version.
+const STORAGE_VERSION: StorageVersion = StorageVersion::new(0);
+
+// We have to temporarily allow some clippy lints. Later on we'll send patches to substrate to
+// fix them at their end.
+#[allow(clippy::missing_docs_in_private_items)]
+#[frame_support::pallet]
+pub mod pallet {
```

Comment:
Previously, we had a convenient way to know how many offences happened on the chain over time. Let's add a simple counter of offences, so that we can still have that metric.

Reply:
https://github.com/humanode-network/humanode/pull/1291/commits/e03ac3da01bb5cefc4049801c73d9e054f8b8a02